### PR TITLE
Updated Java documentation

### PR DESCRIPTION
**Pull request by drodriguezhdez**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ After that, you need to instrument your tests using the **Scope agent** for your
 
 * [.NET agent instructions](dotnet-installation.md)
 * [Java agent instructions](java-installation.md)
+* [Scala agent instructions](java-installation.md)
```
Comment (fermayo) [inline docs/index.md]: I wouldn't add the Scala agent as a different one. New Relic, Datadog, etc. just add Scala as a supported feature of their Java agent. I'd do the same.

File: docs/java-compatibility.md [hunk 2]
```diff
@@ -4,34 +4,65 @@ title: Scope Java Agent compatibility
...
 | [`OpenJDK JVM`](https://openjdk.java.net/)        | `1.7` to `12` |    ✓    |   ✓   |   ✓  |
 | [`Oracle Hotspot JVM`](https://www.oracle.com/technetwork/java/javase/overview/index.html) | `1.7` to `12` |    ✓    |   ✓   |   ✓  |
 
+### Scala versions
+
+The Scope Java agent is compatible with the following Scala versions:
+
+| Scala                | Versions   | Windows | Linux | OS X |
+|--------------------|:----------:|:-------:|:-----:|:----:|
+| [`Scala-lang`](https://www.scala-lang.org/)        | `2.10` to `2.13` |    ✓    |   ✓   |   ✓  |
```
Comment (fermayo) [inline docs/java-compatibility.md]: `Scala-lang` -> `Scala`

File: docs/index.md (hunk 1 above)
Comment (drodriguezhdez) [reply docs/index.md]: Oki. Thanks!

File: docs/java-compatibility.md (hunk 2 above)
Comment (drodriguezhdez) [reply docs/java-compatibility.md]: Oki!

**Pull request merged**